GramAddict/plugins/interact_usernames.py

```diff
@@ -186,12 +189,14 @@ def interact():
                     == FollowingStatus.NOT_IN_LIST
                 )
 
-                interaction_succeed, followed = interaction(
+                interaction_succeed, followed, scraped = interaction(
                     device, username=username, can_follow=can_follow
                 )
-                storage.add_interacted_user(username, followed=followed)
+                storage.add_interacted_user(
+                    username, followed=followed, scraped=scraped
+                )
                 can_continue = on_interaction(
-                    succeed=interaction_succeed, followed=followed
+                    succeed=interaction_succeed, followed=followed, scraped=scraped
                 )
                 if not can_continue:
                     return False
```

Comment:
unrelated to this file, should we also not possibly support scraping a list of people to unfollow? Though, that would need a new plugin built for it, it might be good to do.

Reply:
nice idea!